Formula/vile.rb

```diff
@@ -0,0 +1,29 @@
+class Vile < Formula
+  desc "Vi Like Emacs Editor"
+  homepage "https://invisible-island.net/vile/"
+  url "https://invisible-island.net/archives/vile/current/vile-9.8v.tgz"
+  sha256 "240edec7bbf3d9df48b3042754bf9854d9a233d371d50bba236ec0edd708eed5"
+  license "GPL-2.0-or-later"
+
+  uses_from_macos "flex" => :build
+  uses_from_macos "groff" => :build
+  uses_from_macos "expect" => :test
+  uses_from_macos "ncurses"
+  uses_from_macos "perl"
+
+  def install
+    system "./configure", *std_configure_args,
+                          "--disable-imake",
+                          "--enable-colored-menus",
+                          "--with-ncurses",
+                          "--with-screen=ncurses",
+                          "--without-x"
+    system "make", "install"
+  end
+
+  test do
+    pipe_output("env TERM=xterm expect -",
+      "spawn vile;expect \"unnamed\";send \":w new\r:q\r\";expect eof")
```

Comment:
Let's try to make this easier to read. The `expect` script may need tweaking. We need to do `\\r` to avoid `ruby` expanding the `\r`. (Or maybe it works without it, not sure.)
```suggestion
    ENV["TERM"] = "xterm"
    (testpath/"vile.exp").write <<~EOS
      spawn #{bin}/vile
      expect "unnamed"
      send ":w new\\r:q\\r"
      expect eof
    EOS
    system "expect", "-f", "vile.exp"
```

Reply:
Let me try something like that.  And see why Ubuntu is now failing ... hopefully about done.

Reply:
Should be fixed in https://github.com/Homebrew/homebrew-core/pull/106086/commits/5a52954a6f9b43068761129475d121390bb23d20